metasploit-framework.gemspec

```diff
@@ -47,7 +47,7 @@ Gem::Specification.new do |spec|
   # Database support
   spec.add_runtime_dependency 'activerecord', *Metasploit::Framework::RailsVersionConstraint::RAILS_VERSION
   # Need 3+ for ActiveSupport::Concern
-  spec.add_runtime_dependency 'activesupport', *Metasploit::Framework::RailsVersionConstraint::RAILS_VERSION
+  #spec.add_runtime_dependency 'activesupport', *Metasploit::Framework::RailsVersionConstraint::RAILS_VERSION
```

Comment:
Does this need to be kept in place to enforce all gems provide compatible RAILS versions during weekly `bundle update`?

Reply:
I'll pull it back in. It's our number one memory consumer after all of the low-hanging fruit is eliminated, so this was just an experiment, wasn't meant for this PR yet.